packages/components/src/components/toast/toast/toast-manager.ts

```diff
@@ -64,8 +66,8 @@ export class ToastManagerClass {
       () => forceUpdate(this.toastEl),
       // respect --p-temporary-toast-timeout css variable to override timeout during e2e and vrt tests
       ROLLUP_REPLACE_IS_STAGING === 'production' || process.env.NODE_ENV === 'test'
-        ? ANIMATION_DURATION
-        : parseInt(getComputedStyle(this.toastEl).getPropertyValue('--p-animation-duration'), 10) || ANIMATION_DURATION
+        ? MOTION_DURATION
```

Comment:
Now we lost the connection between styles and js timeouts which could lead to out-of-sync issues in the future.